ddtrace/llmobs/_evaluators/runner.py

```diff
@@ -70,27 +72,34 @@ def recreate(self) -> "EvaluatorRunner":
     def on_shutdown(self):
         self.executor.shutdown()
 
-    def enqueue(self, span_event: Dict) -> None:
+    def enqueue(self, span_event: Dict, span: Span) -> None:
```

Comment:
We add `span` here to the buffer because we need to pass in the `Span` object itself to `SamplingRule.sample`